cms/migrations/0048_convert_richtextfield_to_streamfield.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 2.2.10 on 2020-04-06 11:40
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+from wagtail.core.rich_text import RichText
+
+
+def convert_to_streamfield(apps, schema_editor):
```

Comment:
This is only going to convert the current version of a published page and not update a previous revision, in which case if someone reverts to a previous revision the streamfield will break. Please refer to https://docs.wagtail.io/en/v2.8/topics/streamfield.html#migrating-richtextfields-to-streamfield for instructions on updating all the revisions of published/unpublished pages.

Reply:
@pdpinch  a quick question, do we have any draft Program/Benefit pages over PROD ?